proto/positions.proto

```
@@ -0,0 +1,255 @@
+syntax = "proto3";
+
+package marginoffer.v1;
```

Buf (1.55.1)

Files with package "marginoffer.v1" must be within a directory "marginoffer/v1" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)